src/StatPlots.jl

```diff
@@ -17,7 +17,9 @@ import Loess
 
 export groupapply
```

Comment:
It'd be elegant to reinterpret groupapply in the light of `given`. Intuitively I'd think it could be simplified a lot, but I may be wrong.